src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_WORD = 0;
+    private static final int INDEX_OF_AMOUNT = 1;
+    private static final String STRING_TO_COMPARE = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] fileContent = readFromFile(fromFileName);
+        String report = createReport(fileContent);
+        writeToFile(toFileName, report);
+    }
+
+    public String[] readFromFile(String fileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("can't read file", e);
```

Comment:
```suggestion
            throw new RuntimeException("can't read file " + fileName, e);
```

lets make exception messages more informative

Reply:
fixed